cmd/humioctl/repos_block_ingest.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+)
+
+func newReposBlockIngestCmd() *cobra.Command {
+	var allRepos bool
+
+	cmd := &cobra.Command{
+		Use:   "block-ingest [flags] [repository] <seconds>",
```

Comment:
It might be nicer to have seconds be a flag rather than a positional argument. One reason is that you can define the flag as a `time.Duration` so one could say `--duration=4h`. Another is that it will simplify some of the logic around the positional args.